src/components/BorrowerProfile/LendCta.vue

```diff
@@ -0,0 +1,358 @@
+<template>
+	<span class="tw-z-1">
+		<div class="tw-fixed
+					tw-left-0
+					md:tw-relative
+					md:tw-left-unset
+					tw-bottom-0
+					tw-right-0
+					md:tw-rounded-b
+					md:tw-rounded-t-none
+					lg:tw-rounded-t
+					tw-bg-white
+					tw-border
+					tw-bt-1
+					tw-border-gray-300
+					md:tw-border-none
+					tw-pt-1
+					tw-pl-2.5
+					tw-pr-2.5
+					lg:tw-pl-4
+					lg:tw-pr-4"
```

Comment:
Because the `pl` and `pr` classes are using the same values, they can be combined using `px`, e.g.
```
tw-px-2.5
lg:tw-px-4
```